File: PHP/before.php
<?php
    $a = 0;
    $b = 0;
    $c = 0;
    $d = 0;

    while (true) {
        $a = $a + 1;

        if ($a > 100) {
            break
        }

        $b = $a % 3;
        $c = $a % 5;
        $d = $a % 15;

        if ($d = 0) {
            echo "FizzBuzz", "\n";
            continue
        }

        if ($b = 0) {
            echo "Fizz", "\n";
            continue
        }

        if ($c = 0) {
            echo "Buzz", "\n";
            continue
        }

        echo $a, "\n";
    };
?>

File: Java/before.java
public class Main {
    public static void main(String[] args) {
        int a = 0;
        int b = 0;
        int c = 0;
        int d = 0;

        while(true) {
            a = a + 1;

            if (a > 100) {
                break
            }

            b = a % 3;
            c = a % 5;
            d = a % 15;

            if (d = 0) {
                System.out.println("FizzBuzz");
                continue
            }

            if (b = 0) {
                System.out.println("Fizz");
                continue
            }

            if (c = 0) {
                System.out.println("Buzz");
                continue
            }

            System.out.println(a);
        }
    }
}
